CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ target_include_directories(date_interface INTERFACE
 )
 
 
+install( TARGETS date_prj EXPORT DateConfig )
```

Comment:
Have you tried to generate project with these changes? When I try I get the following error:
```
CMake Error at CMakeLists.txt:96 (install):
  install TARGETS given target "date_prj" which does not exist in this
  directory.
```
There is no target date_prj in this file.

Reply:
Thanks.